src/analysisd/decoders/syscollector.c

```diff
@@ -417,10 +462,13 @@ int decode_netinfo(char *agent_id, cJSON * logJSON,int *socket) {
                 }
 
                 if (address) {
```

Comment:
For IPv6, the gateway and the dhcp status are not being stored in the event structure (lines 448 and 454).